crates/rspack_plugin_javascript/src/plugin.rs

```diff
@@ -138,6 +138,10 @@ impl JsPlugin {
     sources.add(self.render_require(args));
     sources.add(RawSource::from("\n}\n"));
 
+    sources.add(RawSource::from(
+      "__webpack_require__.cache = __webpack_module_cache__;\n",
```

Comment:
directly add `cache` on __webpack_require__ is not a good idea, maybe we should replace `require.cache` to `__webpack_require__.c` as same as webpack

Reply:
Yes, this issue obviously shouldn’t be resolved like this. I will find a solution as soon as possible.